src/io/files.js

```diff
@@ -1357,14 +1419,30 @@ p5.PrintWriter = function(filename, extension) {
    * writer.close();
    * </code>
    * </div>
-   * <div class='norender notest'>
+   *
+   * <div>
    * <code>
-   * // create a file called 'newFile2.txt'
-   * let writer = createWriter('newFile2.txt');
-   * // write some data to the file
-   * writer.write([100, 101, 102]);
-   * // close the PrintWriter and save the file
-   * writer.close();
+   * let writer;
+   * let c; // color
+   * function setup() {
+   *    createCanvas(100, 100);
+   *    background(234, 40, 95);
+   *    button = createButton('GET COLOR');
+   *    button.style('font-size', '8px');
+   *    button.position(21, 35);
+   *    button.size(60, 25);
+   *    button.mousePressed(createFile);
+   * }
+   * function createFile() {
+   *    // create writer object
+   *    let writer = createWriter('newFile.txt');
+   *    // get color at mouseX and Y, which will be the background color
+   *    c = get(mouseX, mouseY);
+   *    // write color into file
+   *    writer.write(c[0] + ", " + c[1] + ", " + c[2]);
+   *    // close writer
+   *    writer.close();
+   * }
```

Comment:
it's great to have a complete example here! but can you add it as a second example and keep the original as well?